pyimager/pyimager.py

```diff
@@ -5,53 +5,59 @@
 from scipy.ndimage.filters import convolve
 
 
-def circropper(input_path, output_path, margin):
+def circropper(input_path, margin):
     """
     Crops an image into a circle and leave some margin as you defined 
 
     Parameters:
     -----------
     input_path: string 
         The file path of the image to be cropped
-    output_path: string
-        The file path for cropped image 
-    margin: float
+    margin: float or int 
         The distance between circle boundary and the original image boundary
     
     Returns:
     --------
-        A numpy array of cropped image
+        A new cropped Image object 
         
     Examples:
     ---------
     >>> from pyimager import circropper
-    >>> circropper("bear.jpg", "result.png", 0)
-    A file named "result.png" will be generated in the 
-    designated folder.
+    >>> circropper("bear.jpg", 0)
+    A Image is cropped to a circle with no margin 
     """
+    # Test argument  
+    if type(input_path) != str and type(margin) != float and type(margin) != int:
+        raise TypeError("The 'input_path' argument must be a string and the margin argument must be a number")
+    if type(input_path) != str:
+        raise TypeError("The 'input_path' argument must be a string")
+    if type(margin) != float and type(margin) != int:
+        raise TypeError("The 'margin' argument must be a float")
+    
+    # Test valid image path 
+    try:
+        Image.open(input_path)
+    except IOError:
+        print ("can't find image")
```

Comment:
I think we generally want to stick to the same assertion/ raise error format as above
```suggestion
    # Test valid image path 
    if os.path.exists(input_path) == False:
        raise FileNotFoundError("The input file does not exist")
```